packages/sdk-v2/src/sdk/ipfs/utils/toV1.ts

```diff
@@ -0,0 +1,3 @@
+import { CID } from "multiformats/cid";
+// eslint-disable-next-line
+export const toV1 = cid => CID.parse(cid).toV1().toString();
```

Comment:
just this one is directly tied with ifs, but it's a small fn so could be exported from the sdk.ts

Reply:
ok